frontend/src/components/pages/acls/Acl.List.tsx

```diff
@@ -593,24 +603,49 @@ const RolesTab = observer(() => {
   );
 });
 
-const AclsTab = observer((p: { principalGroups: AclPrincipalGroup[] }) => {
-  useEffect(() => {
-    void api.refreshAcls(AclRequestDefault, true);
-  }, []);
+const AclsTab = observer((_: { principalGroups: AclPrincipalGroup[] }) => {
+  const { data: principalGroups, isLoading } = useListACLAsPrincipalGroups();
+  const { mutateAsync: deleteACLMutation } = useDeleteAclMutation();
 
   const [aclFailed, setAclFailed] = useState<{ err: unknown } | null>(null);
   const [editorType, setEditorType] = useState<'create' | 'edit'>('create');
   const [edittingPrincipalGroup, setEdittingPrincipalGroup] = useState<AclPrincipalGroup | null>(null);
 
-  let groups = p.principalGroups.filter((g) => g.principalType === 'User');
+  const nav = useNavigate();
```

Comment:
nit; should we try to be consistent and use `const navigate = useNavigate()` like elsewhere and also in react-router docs?

Reply:
make sense, done